src/main.css

```diff
@@ -7,4 +7,5 @@ html, body, #root {
 
 #root {
     padding: 0px 8px;
```

Comment:
I'm afraid we need to remove the padding/margin on the admin side and add them to the agent side.

So, here we need to remove everything `margin: 0; padding: 0;`

and all pages in the agent side should be wrapped in a container with `margin: 8px`

Reply:
Here's an example of the container in Asana app where we add margins where necessary
https://github.com/DeskproApps/asana/blob/master/src/components/LinkTasks/LinkTasks.tsx#L58

Reply:
As discussed, it's not a regression (as that padding already exists) and is minor enough for it to not be a blocker for merging. We can always revisit.